tests/scripts/test_send_tasks.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see<http://www.gnu.org/licenses/>
+
+
+import unittest
+import os
+from unittest.mock import patch, MagicMock
+from pathlib import Path
+from lxml import etree
+from . import GmpMockFactory, load_script
+
+
+CWD = os.path.abspath(os.path.join(__file__, '../'))
+
+
+class SendTasksTestCase(unittest.TestCase):
+    def setUp(self):
+        self.send_tasks = load_script(
+            os.path.join(CWD, '../../scripts'), 'send-tasks'
```

Comment:
pathlib is the better choice for handling this.

Reply:
Tell that to python3.5 ;)